spec/features/create_user_spec.rb

```diff
@@ -18,6 +18,13 @@
     it "sends them to the dashboard" do
       expect(current_path).to eq "/dashboard"
     end
+    
+    it "removes the sign in and sign up links, replaces them with logout" do
```

Comment:
LOVE that you're testing this. Maybe move it to spec/features/create_session_spec.rb, then you can test signing out of a session as well! :smile: